frontend/src/formdescriptioneditor/RadioWidget.tsx

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import { WidgetProps } from 'formdescriptioneditor/WidgetEntry.types';
+import React, { useEffect, useRef, useState } from 'react';
+
+const useStyles = makeStyles(() => ({
+  selected: {
+    color: 'var(--blue-lagoon)',
```

Comment:
use `theme.palette.primary.main` instead of 'var(--blue-lagoon)'

Reply:
Fixed